frontend/src/__tests__/cypress/cypress/tests/mocked/projects/workbench.cy.ts

```diff
@@ -399,10 +400,72 @@ describe('Workbench page', () => {
     });
 
     cy.get('@editWorkbench.all').then((interceptions) => {
-      expect(interceptions).to.have.length(2); // 1 dry run request and 1 actaul request
+      expect(interceptions).to.have.length(2); // 1 dry run request and 1 actual request
     });
   });
 
+  it('Handle deleted notebook sizes in workbenches table', () => {
+    initIntercepts({});
+    cy.interceptOdh(
+      'GET /api/config',
+      mockDashboardConfig({
+        notebookSizes: [
+          {
+            name: 'Medium',
+            resources: {
+              limits: {
+                cpu: '6',
+                memory: '24Gi',
+              },
+              requests: {
+                cpu: '3',
+                memory: '24Gi',
+              },
+            },
+          },
+          {
+            name: 'Large',
+            resources: {
+              limits: {
+                cpu: '14',
+                memory: '56Gi',
+              },
+              requests: {
+                cpu: '7',
+                memory: '56Gi',
+              },
+            },
+          },
+          {
+            name: 'X Large',
+            resources: {
+              limits: {
+                cpu: '30',
+                memory: '120Gi',
+              },
+              requests: {
+                cpu: '15',
+                memory: '120Gi',
+              },
+            },
+          },
+        ],
+      }),
+    );
+    workbenchPage.visit('test-project');
+    const notebookRow = workbenchPage.getNotebookRow('Test Notebook');
+    notebookRow.shouldHaveNotebookImageName('Test Image');
+    notebookRow.shouldHaveContainerSize('Custom');
+    notebookRow.findKebabAction('Edit workbench').click();
+    editSpawnerPage.shouldHaveContainerSizeInput('Custom');
+    workbenchPage.visit('test-project');
+    workbenchPage.findCreateButton().click();
+    createSpawnerPage.findSubmitButton().should('be.disabled');
+    verifyRelativeURL('/projects/test-project/spawner');
+    // Custom container size dropdown option should not be present for create workbench
+    createSpawnerPage.shouldNotHaveCustomContainerSizeInput();
```

Comment:
Ah -- your usage here seems weird. Why not support the inverse of the usage a few lines above
```suggestion
    createSpawnerPage.shouldNotHaveContainerSizeInput('Custom');
```

Reply:
I'm doing this: ` createSpawnerPage.findContainerSizeInput('Keep custom size').should('not.exist');`